tools/cloud-build/daily-tests/tests/slinky.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+test_name: slinky
+deployment_name: slinky-{{ build }}
+region: us-central1
+zone: us-central1-a
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/community/examples/hpc-slinky/hpc-slinky.yaml"
+network: "{{ deployment_name }}-net"
+remote_node: "{{ deployment_name }}-0"
+cli_deployment_vars:
+  authorized_cidr: "{{ build_ip.stdout }}/32"
+  gcp_public_cidrs_access_enabled: true
+post_deploy_tests: []
```

Comment:
We would need some tests to be run post deployment. I don't think this test would be executed.
Please refer to the following code pointers:
- [Build changes to run daily tests](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/tools/cloud-build/daily-tests/builds/gke-a4.yaml): We would need a similar file in the builds folder similar to this one to run the test.
- [Integration test run for A4 machine family](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/tools/cloud-build/daily-tests/tests/gke-a4.yml#L41): Please check if the test highlighted here would work for slinky. If not, you would have to define a similar test for slinky.

Reply:
Thanks @ighosh98. Added the build config and a new test-validation (none of the existing ones really fit the bill). I was originally envisioning a simple deploy/destroy test without much application-level validation like [GKE test](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/057bfe21bc93e9e6fd13ed4fd37a8130000896bb/tools/cloud-build/daily-tests/tests/gke.yml#L24), but in hindsight definitely agree with the deeper validation.

Of course, it's a bit tricky to develop/debug these integration tests without the ability to run them, but I suspect I'm close. Thanks for the guidance on this.